.github/workflows/test_installation.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
       needs: build_package
       runs-on: ubuntu-latest
       strategy:
+        fail-fast: false
         matrix:
           python-version: [ "3.7", "3.8", "3.9", "3.10" ]
```

Comment:
it will now break on 3.7, right?

Reply:
There is some issue however with building cli

Reply:
Yes, I removed it from the text matrix